templates/repo/commit_load_branches_and_tags.tmpl

```diff
@@ -1,3 +1,4 @@
+{{if not .PageIsWiki}}
```

Comment:
Yes, that's one approach.
Another would be to show the information for the wiki repo when present.
The only thing we would need to change in that case is line 4.

Reply:
(and add an additional route for the wiki repo)

Reply:
Which is better?
IMO, wiki's branches and tags are generated by Gitea, and should not show them to the end user.

Reply:
emmm, it seems that we can clone it, and push changes to it. So maybe display the correct info is better.
![image](https://github.com/go-gitea/gitea/assets/18380374/79fa5e60-76d9-4a6c-8714-a676dcf97322)